apps/web/src/state/multicall/hooks.ts

```diff
@@ -235,7 +235,8 @@ export function useSingleContractMultipleData<TAbi extends Abi | readonly unknow
   const calls = useMemo(
     () =>
       contract && contract.abi && contract.address && args && args.length > 0
-        ? args.map<Call>((inputs) => {
+        ? args.map((inputs) => {
```

Comment:
Do we need to add type assertion `as Call[]` at the end of the map call if we remove the generic `<Call>` here?